rust-src/wallet_library/src/statement.rs

```diff
@@ -0,0 +1,375 @@
+use concordium_base::{
+    base::ContractAddress,
+    common::Serialize,
+    curve_arithmetic::Curve,
+    id::{id_proof_types::AtomicStatement, types::*},
+    web3id::{CredentialStatement, Request},
+};
+use std::{
+    collections::{BTreeMap, HashSet},
+    fmt::Display,
+    hash::Hash,
+    marker::PhantomData,
+};
+
+pub trait AcceptableRequest<C: Curve, AttributeType: Attribute<C::Scalar>> {
```

Comment:
Please document the purpose of the trait

Reply:
I've tried to do so. :upside_down_face: